integration-tests/typescript-angular/src/generated/api.github.com.yaml/client.service.ts

```diff
@@ -320,8 +320,44 @@ import { HttpClient, HttpParams, HttpResponse } from "@angular/common/http"
 import { Injectable } from "@angular/core"
 import { Observable } from "rxjs"
 
+export type GitHubV3RestApiServiceServer =
+  Server<"GitHubV3RestApiServiceServer">
+
+export class GitHubV3RestApiServiceServers {
+  static default(): GitHubV3RestApiServiceServer {
```

Comment:
Is it clear enough that `default` / `specific` are for the main base URL? Hopefully the branded types make this clear, but could consider separating them from the operation specific stuff